plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -90,6 +90,14 @@ func (r *Resource) AddOpaque(name v1.ResourceName, quantity int64) {
 	r.OpaqueIntResources[name] += quantity
 }
 
+func (r *Resource) SetOpaque(name v1.ResourceName, quantity int64) {
```

Comment:
nit: to avoid duplication of lazy init i think `AddOpaque` should only do the calculation for adding and then call `SetOpaque`?